flexboxAPIspecV2.md

```diff
@@ -0,0 +1,783 @@
+```
+
+
+
+## Flex Grow
+
+The flex-grow property specifies how much the item will grow relative to the rest of the flexible items inside the same container.
+
+Note: If the element is not a flexible item, the flex-grow property has no effect.
+
+
+```xml
+ <controls:LayoutPanel MinWidth="500">
+            <controls:LayoutPanel.Layout>
+                <controls:FlexboxLayout/>
+            </controls:LayoutPanel.Layout>
+            <Grid controls:FlexboxLayout.Grow="0" controls:FlexboxLayout.Basis="100" >
+                <TextBlock>0</TextBlock>
+            </Grid>
+            <Grid controls:FlexboxLayout.Grow="1" controls:FlexboxLayout.Basis="100">
+                <TextBlock>1</TextBlock>
+            </Grid>
+            <Grid controls:FlexboxLayout.Grow="0" controls:FlexboxLayout.Basis="100">
+                <TextBlock>0</TextBlock>
+            </Grid>
+        </controls:LayoutPanel>
+```
+
+### Output of the above code (styling omitted)
+![image](./FlexGrow.PNG)
+
+
+##  Flex Grow Accepted Values
+
+
+
+| Value | Description |
+|-|-|
+| Number (Default Value = 0) | Specifies how much the item will grow relative to the rest of the flexible items. |
+
+# API Details
+
+
+```c# (but really MIDL3)
+
+unsealed runtimeclass FlexboxLayout : NonVirtualizingLayout
+    
+    {
+        static Windows.UI.Xaml.DependencyProperty GrowProperty{ get; };
+        static void SetGrow(Windows.UI.Xaml.UIElement element, Double value);
+        static Double GetGrow(Windows.UI.Xaml.UIElement element);
+    }
+```
+
+
+
+
+
+## Flex Shrink
+
+The flex-shrink property specifies how the item will shrink relative to the rest of the flexible items inside the same container.
+
+Note: If the element is not a flexible item, the flex-shrink property has no effect.
+
+
+```xml
+<controls:LayoutPanel Width="500">
+    <controls:LayoutPanel.Layout>
+          <controls:FlexboxLayout/>
+    </controls:LayoutPanel.Layout>
+            <Grid controls:FlexboxLayout.Shrink="1" Width=" 250">
+                <TextBlock>1</TextBlock>
+            </Grid>
+            <Grid controls:FlexboxLayout.Shrink="1" Width=" 250">
+                <TextBlock>1</TextBlock>
+            </Grid>
+            <Grid controls:FlexboxLayout.Shrink="1" Width=" 250">
+                <TextBlock>1</TextBlock>
+            </Grid>
+</controls:LayoutPanel>
+```
+
+### Output of the above code (styling omitted)
+
+todo
+
+
+##  Flex Shrink Accepted Values
+
+
+
+| Value | Description |
+|-|-|
+| Number (Default Value = 1) | Specifies how much the item will shrink relative to the rest of the flexible items. |
+
+# API Details
+
+
+```c# (but really MIDL3)
+
+unsealed runtimeclass FlexboxLayout : NonVirtualizingLayout
+    
+    {
+         static Windows.UI.Xaml.DependencyProperty ShrinkProperty{ get; };
+         static void SetShrink(Windows.UI.Xaml.UIElement element, Double value);
+         static Double GetShrink(Windows.UI.Xaml.UIElement element);
+    }
+```
+
+
+
+
+
+## Flex Basis
```

Comment:
This could use some more explanation. Might be enough though to just add the explanatory text on the code sample. 

There should be an explicit mention that this is based on Basis and Unit.